hw/bsp/olimex-pic32-hmz144/pkg.yml

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+pkg.name: hw/bsp/olimex-pic32-hmz144
+pkg.type: bsp
+pkg.description: BSP definition for the Olimex PIC32-HMZ144 board.
+pkg.author: "Apache Mynewt <dev@mynewt.apache.org>"
+pkg.homepage: "http://mynewt.apache.org/"
+pkg.keywords:
+    - pic32
+    - microchip
+
+pkg.cflags:
+    - -mprocessor=32MZ2048EFM144
+    - -G6
+pkg.lflags:
+    - "-L@apache-mynewt-core/hw/bsp/olimex-pic32-hmz144"
```

Comment:
Weird, why does the BSP home have to be manually added here?

Reply:
This one requires some explanation:
linker script allow only one **MEMORY** section and PIC32 has this part very long.
PIC32MZ MCUs differ very little in memory sizes to make part that is different small
I decided to include fragment of MEMORY section in statement `INCLUDE p32mz_app_mem.ld`
but this requires additional **pkg.lflags** to access BSP part of linker script.
In ARM based linker scripts MEMORY section is very short and is added as separate linker script.

Reply:
Makes sense, thanks!